core/templates/css/oppia.css

```diff
@@ -4166,6 +4167,10 @@ oppia-noninteractive-link {
   text-align: center;
 }
 
+.participation h2:not(:first-child) {
+  margin-top: 2.5em;
+}
```

Comment:
@kevintab95 @apruvbotle is this something that is global or specific to certain pages? If it is specific to certain pages, how many pages?

Reply:
Looks specific. If so, @apurvbotle can you please add it to the relevant component template (nice catch @srijanreddy98!). Thanks!

Reply:
@srijanreddy98 @kevintab95 Done. Thanks for pointing out.